README.md

```diff
@@ -27,6 +28,20 @@ Its a recipe app website.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+### Key Features <a name="key-features"></a>
+
+- **Add Category**
+- **Add Transaction**
+- **View Transaction**
+
+## 🚀 Live Demo <a name="live-demo"></a>
```

Comment:
- [ ] Kindly consider making a presentation demo for your project which highlights all the project features and also highlights a piece of code that you are proud of it.
> Not: The link you submitted in the PR description, is for another activity and about 1 month ago